Utils/handlerutil2.py

```diff
@@ -131,20 +132,30 @@ def _get_current_seq_no(self, config_folder):
         seq_no = -1
         cur_seq_no = -1
         freshest_time = None
-        for subdir, dirs, files in os.walk(config_folder):
-            for file in files:
-                try:
-                    cur_seq_no = int(os.path.basename(file).split('.')[0])
-                    if (freshest_time == None):
-                        freshest_time = os.path.getmtime(join(config_folder, file))
-                        seq_no = cur_seq_no
-                    else:
-                        current_file_m_time = os.path.getmtime(join(config_folder, file))
-                        if (current_file_m_time > freshest_time):
-                            freshest_time = current_file_m_time
-                            seq_no = cur_seq_no
-                except ValueError:
-                    continue
+
+        # First read the sequence number from the environment variable
+        seq_no_from_env = os.getenv(ENV_CONFIG_SEQUENCE_NUMBER)
+        if (seq_no_from_env is not None):
+            seq_no = int(seq_no_from_env)
```

Comment:
What if the env variable existed, but isn't in the format that can be parsed into int?

Catch ValueError and/or TypeError